src/blockchain_db/blockchain_db.cpp

```diff
@@ -389,8 +389,6 @@ void BlockchainDB::fixup()
     LOG_PRINT_L1("Database is opened read only - skipping fixup check");
     return;
   }
-
-  set_batch_transactions(true);
```

Comment:
No longer necessary because 
https://github.com/loki-project/loki/blob/fa9adfc4433e9668477a231899759e741e3f0509/src/blockchain_db/lmdb/db_lmdb.h#L169
ensures batch transaction is set. Leaving it here causes extra warnings on startup.